amlip_docs/rst/demo/wan.rst

```diff
@@ -0,0 +1,189 @@
+.. include:: ../exports/alias.include
```

Comment:
The new files are not included in any doctree, and thus they are not generated in the documentation